content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/clientlibs/editor/js/title.js

```diff
@@ -0,0 +1,233 @@
+/*******************************************************************************
```

Comment:
Does it duplicate `title.js` of Title v2? If so, we could remove it and use the v2 client lib (see comment above)